SimMuon/GEMDigitizer/src/ME0ReDigiProducer.cc

```diff
@@ -128,8 +131,9 @@ void ME0ReDigiProducer::buildDigis(const ME0DigiPreRecoCollection & input_digis,
       if (reDigitizeOnlyMuons_ and fabs(me0Digi.pdgid()) != 13) continue;
       if (!reDigitizeNeutronBkg_ and !me0Digi.prompt()) continue;
 
-      // scale for luminosity
-      if (CLHEP::RandFlat::shoot(engine) > instLumi_*1.0/10) continue;
+      // scale background hits for luminosity
+      if (!me0Digi.prompt() or fabs(me0Digi.pdgid()) != 13)
+	if (CLHEP::RandFlat::shoot(engine) > instLumi_*1.0/5) continue;
```

Comment:
instLumi should be set to 5 infact in the configuration, as it is the code can only throw away hits